src/components/dataset/schemaCreation/DatasetSchemaBuilderView.tsx

```diff
@@ -0,0 +1,780 @@
+      </div>
+
+      <div className={classes.schemaSectionHeader}>
+        <Typography variant="h3">Schema for:</Typography>
+        <div className={classes.schemaSectionHeaderDesc}>{getValues('name')}</div>
+      </div>
+
+      <div className={classes.schemaBuilderContainer}>
+        <Typography variant="h4">Schema</Typography>
+        <div className={classes.schemaBuilderMain}>
+          <div className={classes.schemaBuilderStructureView}>
+            <div className={classes.schemaBuilderStructureViewControls}>
+              <div>
+                <Button
+                  id="schemabuilder-createTable"
+                  color="primary"
+                  disableElevation
+                  variant="contained"
+                  type="button"
+                  className={classes.schemaControlButton}
+                  onClick={createTable}
+                >
+                  <AddCircleRounded className={classes.iconInButton} />
+                  Create a table
+                </Button>
+
+                <Button
+                  id="schemabuilder-createColumn"
+                  color="primary"
+                  disableElevation
+                  variant="contained"
+                  type="button"
+                  disabled={selectedTable === -1}
+                  className={classes.schemaControlButton}
+                  onClick={createColumn}
+                >
+                  <AddCircleRounded className={classes.iconInButton} />
+                  Create a column
+                </Button>
+              </div>
+
+              <div className={classes.flexRow}>
+                <IconButton
+                  id="datasetSchema-up"
+                  size="small"
+                  color="primary"
+                  className={classes.iconButton}
+                  disabled={selectedTable === -1}
+                  onClick={() => {
+                    const schemaCopy = _.cloneDeep(datasetSchema);
+                    if (selectedColumn > 0) {
+                      swapArrayLocs(
+                        schemaCopy.tables[selectedTable].columns,
+                        selectedColumn,
+                        selectedColumn - 1,
+                      );
+                      setSelectedColumn(selectedColumn - 1);
+                    } else if (selectedColumn === -1 && selectedTable > 0) {
+                      swapArrayLocs(schemaCopy.tables, selectedTable, selectedTable - 1);
+                      setSelectedTable(selectedTable - 1);
+                    }
+                    setDatasetSchema(schemaCopy);
+                  }}
+                >
+                  <i className="fa fa-angle-up" />
+                </IconButton>
+
+                <IconButton
+                  id="datasetSchema-down"
+                  size="small"
+                  color="primary"
+                  className={classes.iconButton}
+                  disabled={selectedTable === -1}
```

Comment:
There are other cases when we could disable this, like when the column is the last in the last table, etc. But I wonder if it's worth the more complicated logic.

Reply:
I have a unit test that confirms that clicking the button when the col is already the last in the list doesnt do anything - We could disable the button instead too!